interfaces/builtin/mpris.go

```diff
@@ -88,6 +88,31 @@ dbus (receive)
     bus=session
     path=/org/mpris/MediaPlayer2
     peer=(label=@{profile_name}),
+
+# Allow unconfined session to interact with the player
+dbus (receive)
+    bus=session
+    interface=org.freedesktop.DBus.Properties
+    path=/org/mpris/MediaPlayer2
+    peer=(label=unconfined),
+
+dbus (receive)
```

Comment:
this is listed in mprisConnectedSlotAppArmorClassic, perhaps it no longer needs to be there